analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/LocksShouldBeReleasedTest.cs

```diff
@@ -51,6 +51,18 @@ public class LocksShouldBeReleasedTest
         public void LocksShouldBeReleased_ReaderWriterLockSlim_CS() =>
             Verifier.VerifyAnalyzer(@"TestCases\SymbolicExecution\Sonar\LocksShouldBeReleased.ReaderWriterLockSlim.cs", new LocksShouldBeReleased());
 
+        [TestMethod]
+        [TestCategory("Rule")]
+        public void LocksShouldBeReleased_Mutex_CS() =>
+            Verifier.VerifyAnalyzer(@"TestCases\SymbolicExecution\Sonar\LocksShouldBeReleased.Mutex.cs", new LocksShouldBeReleased());
```

Comment:
This rule will only be implemented for Roslyn-based CFG. All files should be in the `Roslyn` subdirectory. For old rules (after few more MMFs) the `Sonar` directory will hold all test cases for the old engine (no C# 8 for example). `Roslyn` will hold test cases for the new engine (with different FPs/FNs, C# 8+ syntax)

cc @csaba-sagi-sonarsource as the issue comes from the base branch.
```suggestion
            Verifier.VerifyAnalyzer(@"TestCases\SymbolicExecution\Roslyn\LocksShouldBeReleased.Mutex.cs", new LocksShouldBeReleased());
```

Reply:
I suggest not ti fix it in this PR. After it gets merged to the "feature" branch, I`ll fix it there.

Reply:
OK

Reply:
Thanks @csaba-sagi-sonarsource . Indeed, it will be easier to do that after merging on Csaba's branch, to avoid more rebase conflicts.